app/controllers/application_controller.rb

```diff
@@ -15,6 +15,7 @@ class ApplicationController < ActionController::Base
   include ActionView::Helpers::DateHelper
   include ActionView::Helpers::TextHelper
   include CommentsHelper
+  include StoriesHelper
```

Comment:
Is it necessary to include StoriesHelper here if it's being included in more specific controllers?

Reply:
Thanks for the feedback @julianguyen. You are right, in this case we do not need to include `StoriesHelper` in `ApplicationController` since the method is not used in views (unlike `CommentsHelper` method), I've updated the code to avoid including `StoriesHelper` in `ApplicationController` and use a `FakeStoriesController` in specs to avoid depending on a specific controller.

Feedback welcome